src/lib.rs

```diff
@@ -32,6 +32,9 @@
 extern crate unicase;
 extern crate quoted_string;
 
+#[cfg(feature = "serde")]
+extern crate serde;
```

Comment:
I wonder if it's possible to use `extern crate serde` directly in `serde_impl` mod.